From 954ef6ce80838b7e96cfc7f43287e99238c9639c Mon Sep 17 00:00:00 2001
From: Jan Grulich <jgrulich@redhat.com>
Date: Tue, 4 Aug 2026 16:13:26 +0200
Subject: [PATCH] Fix sassc requirement for tarball builds

The meson build checks for base.css to determine if pre-generated CSS
is available, but the tarball ships gtk.css instead. This causes sassc
to be required even for tarball builds where the compiled stylesheet
is already present. Fix the check to look for gtk.css.
---
 src/stylesheet/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/stylesheet/meson.build b/src/stylesheet/meson.build
index 023ebf7b7..1acc22ee4 100644
--- a/src/stylesheet/meson.build
+++ b/src/stylesheet/meson.build
@@ -3,7 +3,7 @@ fs = import('fs')
 stylesheet_deps = []
 
 # For git checkouts, but not for tarballs...
-if not fs.exists('base.css')
+if not fs.exists('gtk.css')
   sassc = find_program('sassc', required: false)
   if not sassc.found()
     subproject('sassc', default_options: ['warning_level=0', 'werror=false'])
-- 
GitLab

